cpp11test/src/test-protect.cpp

```diff
@@ -37,6 +37,13 @@ context("unwind_protect-C++") {
     UNPROTECT(1);
   }
 
+  test_that("stop throws an unwind_exception") {
+    expect_error_as(cpp11::stop("error"), cpp11::unwind_exception);
+    expect_error_with(cpp11::stop("error"), "error");
```

Comment:
```suggestion
    expect_error(cpp11::stop("error"));
```

Reply:
`expect_error_with` checks the error message as well; IIUC `expect_error` just checks that it throws?

Reply:
Never mind, I'll need to open a PR to testthat to add it